### DISCUSSION TITLE
Airflow 2.7.3 UI unstable while running multiple dags

### DISCUSSION THREAD

**Discussion by utkarsh-saama** [apache/airflow#39605] [Q&A]
We recently upgraded airflow from 2.6.2 to 2.7.3. We are facing the below performance issues, mainly on Airflow UI end.

1. Airflow UI restarts frequently while running multiple dags at a time. We have observed that airflow-web pod was showing several restarts while multiple dags running in backend
2. All DAGs are getting displayed in UI as per table "serialized_dags". While running multiple dags at a time, all dags listed in serialized_dags getting disappeared hence, they disappear from UI

Requesting suggestions for resolving the same.

**Comment by tshrinivasan** [comment]
https://www.restack.io/docs/airflow-knowledge-airflow-dag-not-showing

https://forum.astronomer.io/t/dags-are-not-showing-in-my-webserver-ui/1053/2

check the points mentioned here.

check the scheduler log and ui logs for any clue.
